src/sentry/static/sentry/app/actionCreators/environments.jsx

```diff
@@ -8,8 +8,8 @@ export function clearActiveEnvironment() {
   EnvironmentActions.clearActive();
 }
 
-export function loadEnvironments(data) {
-  EnvironmentActions.loadData(data);
+export function loadEnvironments(data, envName) {
```

Comment:
Why combine the two actions here? Isn't there a scenario where you would want to set active environment independent of loading?

Reply:
Yes, there's a separate actions for that (`loadActiveEnvironments` and `loadHiddenEnvironments`), this one is only called on initial load

Reply:
Ah I see. Not needed in this PR, but in general, I think it's better to not have stores fire actions. This would help align us with a 1 way data flow.